### DISCUSSION TITLE
how to display more than 7 rows in aggrid based table?

### DISCUSSION THREAD

**Discussion by cychong47** [zauberzeug/nicegui#944] [Q&A] [answered]
### Question

Hi, 
I have created a table to display with aggrid but the table size is fixed only to display 7 rows.
This is a snippet of code

```python
column_defs = ...
grid = ui.aggrid({"colmDefs": columns_defs, "rowData": [] })

# get data 
with ui.row().classes("max-h-full"):
    for item in data:
        grid.options["rowData"].append(item)
    grid.update()
```

I have changed `max-h-full` to something else such as `max-h-400` but it does not help.

Looks like the aggrid itself does not have any property about the number of row to be displayed.

How can I make more rows (all if possible) to be displayed?

**Comment by falkoschindler** [comment, marked as answer]
You can
- choose a TailwindCSS class for the [height](https://tailwindcss.com/docs/height) of the grid
  ```py
  ui.aggrid({}).classes('h-96')
  ```
- use a TailwindCSS class with an arbitrary height value like this
  ```py
  ui.aggrid({}).classes('h-[500px]')
  ```
- or simply use plain CSS
  ```py
  ui.aggrid({}).style('height: 500px')
  ```

**Comment by cychong47** [comment]
Thanks it works like a charm!
